src/common/imageio_rawspeed.cc

```diff
@@ -120,7 +120,7 @@ dt_imageio_retval_t dt_imageio_open_rawspeed(dt_image_t *img, const char *filena
 
 #ifdef __WIN32__
   const size_t len = strlen(filename) + 1;
-  wchar_t filen[len];
+  wchar_t filen[MAX_PATH];
   mbstowcs(filen, filename, len);
```

Comment:
http://en.cppreference.com/w/cpp/string/multibyte/mbstowcs
len	-	number of wide characters available in the array pointed to by dst

So it should be more like
```
mbstowcs(filen, filename, sizeof(filen) / sizeof(wchar_t));
```